components/coding/profileV2/AssignmentSectionComponent.tsx

```diff
@@ -0,0 +1,87 @@
+import { useQuery } from "@apollo/client";
+import {
+  CheckCircleIcon,
+  PencilAltIcon,
+  XCircleIcon,
+} from "@heroicons/react/outline";
+import Link from "next/link";
+import React, { useState } from "react";
+import { FETCH_ALL_USER_ASSIGNMENTS } from "../../../graphql/fetchAllUserAssignments";
+import {
+  FetchUserAssignmentSubmissionsDataResponse,
+  UserAssignmentSubmissionsData,
+} from "../../../graphql/fetchUserAssignmentSubmissions";
+import { User } from "../../../graphql/fetchUserProfile";
+import { useAuth } from "../../../lib/authContext";
+import ExpandableContainer from "../ExpandableContainer";
+
+export type AssignmentSectionComponentProps = {};
+
+export default function AssignmentsSection({}: AssignmentSectionComponentProps) {
+  const { user } = useAuth();
+  const [userAssignments, setUserAssignments] = useState<
+    UserAssignmentSubmissionsData[]
+  >([]);
+  const { loading: userAssignmentsLoading } =
+    useQuery<FetchUserAssignmentSubmissionsDataResponse>(
+      FETCH_ALL_USER_ASSIGNMENTS,
+      {
+        variables: {
+          user_id: user.uid,
+        },
+
+        onCompleted: (data: FetchUserAssignmentSubmissionsDataResponse) => {
+          setUserAssignments(data.user_assignment_submissions);
+        },
+      }
+    );
+  return (
+    <ExpandableContainer open={true} title={""}>
+      <div>
+        {userAssignments.length > 0 && (
+          <div className="grid grid-cols-5 text-sm font-semibold text-center border-b-2 md:grid-cols-12 md:text-lg">
+            <p className="col-span-2 md:col-span-5">Assignment</p>
+            <p className="hidden md:block md:col-span-2">Completed</p>
```

Comment:
how about we have one "status" column instead of two separate columns, then display the check mark if completed, and the x icon if waiting review... 

design nitpick, but wondering if there's a different icon we can use for whether an assignment is awaiting review